src/main/scala/org/scalatra/sbt/DistPlugin.scala

```diff
@@ -96,18 +119,20 @@ object DistPlugin extends Plugin {
       HiddenFileFilter || PatternFileFilter(".*/WEB-INF/classes") || PatternFileFilter(".*/WEB-INF/lib")
       // could use (webappDest in webapp).value.getCanonicalPath instead of .*, but webappDest is a task and SBT settings cant depend on tasks
     },
-    target in Dist <<= (target in Compile)(_ / "dist"),
-    assembleJarsAndClasses in Dist <<= assembleJarsAndClassesTask,
-    stage in Dist <<= stageTask,
-    dist in Dist <<= distTask,
-    dist <<= dist in Dist,
-    name in Dist <<= name,
-    runScriptName in Dist <<= name,
+    target in Dist := (target in Compile)(_ / "dist").value,
+    assembleJarsAndClasses in Dist := assembleJarsAndClassesTask.value,
+    stage in Dist := stageTask.value,
+    dist in Dist := distTask.value,
+    dist := (dist in Dist).value,
+    name in Dist := name.value,
+    runScriptName in Dist := name.value,
     mainClass in Dist := Some("ScalatraLauncher"),
     memSetting in Dist := "1g",
     permGenSetting in Dist := "128m",
     envExports in Dist := Seq(),
-    javaOptions in Dist <++= (memSetting in Dist, permGenSetting in Dist) map { (mem, perm) =>
+    javaOptions in Dist ++= {
+      val mem = memSetting in Dist
+      val perm = permGenSetting in Dist
```

Comment:
Oh, thank you.

No compile error occurred and I did not notice.
I modified and committed.